spec/javascripts/details/views/notification_bar_spec.js.coffee

```diff
@@ -13,30 +14,119 @@ describe 'Notification', ->
       expect(notification instanceof Notifications).to.eql(true)
   
   describe '#_postNotification', ->
-    it "create a notification", ->
+    it "create a notification without type", ->
+      notification = new Notifications(getNotificationContent())  
+      expect(notification.el.find('ul li').length).to.eql(0)
+      notice = notification._prepare({type: "", content: "This is content", title: ""})
+      notification._postNotification(notice)
+      expect(notification.el.find('ul li').length).to.eql(1)
+      expect(notification.el.find('ul li.notice-type').length).to.eql(0)
+      expect(notification.el.find('ul li.notice-type-primary').length).to.eql(1)
+      expect(notification.el.find('ul li .notification-content').text()).to.eql("This is content")
+    it "create a notification with type", ->
```

Comment:
Line breaks in between examples would help readability here, especially since this file is quite dense.